README.md

```diff
@@ -1,46 +1,58 @@
-# Shared Travis Configuration Imports
+# Auto-updated Node.js version matrix in Travis CI
 
 - This repository is managed by the [Package Maintenance Working Group](https://github.com/nodejs/package-maintenance), see [Governance](https://github.com/nodejs/package-maintenance/blob/master/Governance.md).
 - Read more about [Shared Build Configurations in Travis CI](https://docs.travis-ci.com/user/build-config-imports/).
 
-## Usage
-
-Replace the list of `node_js` versions in your `.travis.yml` with one of the available files.
+Import configurations from this repository into your `.travis.yml` to automatically start testing in new versions of Node.js as they get released.
 
-### Examples
+## Usage
 
-#### `lts` policy: use LTS and current Node.js releases, starting with v10.x 
+In your `.travis.yml`, replace the `node_js` section with the following:
 
 ```
 import:
   - nodejs/ci-config-travis:lts/gte-10.yml
+  #                          ^       ^
+  #                          |       |
+  #                          |       ·---------- Miminum version
+  #                          ·------------------ Upgrade policy
+  # i.e. include all LTS Node.js releases, starting with version 10.x
 ```
 
-- Travis CI will use the latest version of each release line in the list
-- New major Node.js versions, greater or equal to v10.0.0, will be added to the list as soon as they are released
-- Once added, LTS versions will never be removed. Non-LTS versions will be removed when they reach their lifetime.
-    - Note that if your policy is to only support LTS versions, then removing the non-LTS version in your test matrix is not a breaking change, as the non-LTS version was never supported (it was only used for test purposes).
+Depending on the support policy of your library/application, you can choose when and which versions get added to your test matrix.
 
+Once added to the list, the Long Term Support (LTS) versions will never get removed from the list, i.e. you will need to take explicit action to change the minimum version you're importing (note: when you increase the minimum version requirements, you should also bump the major version of your package, to indicate breaking changes according to [semver](https://semver.org/)).
 
-#### `all` policy: use all Node.js releases, starting with v14.x 
+This repository offers three upgrade policies:
 
-```
-import:
-  - nodejs/ci-config-travis:all/gte-14.yml
-```
+- **[`all`](./all)** - new major releases get added as they are released, they never get removed
+- **[`lts`](./lts)** - new major releases get added as they are released, non-LTS releases get removed when their support lifetime ends, LTS versions never get removed
+- **[`lts/strict`](./lts/strict)** - new major releases get added as they reach LTS status, they never get removed   
```

Comment:
Wondering if there is value in adding support for the other aliases?

Reply:
I considered that, but they'd have identical contents with the ones here, given the "we won't remove versions" guarantee.